### PR TITLE
Add keep alive

### PR DESCRIPTION
**Pull request by phillipivan**
- Add keep alive (Vicreo logs wrong type, but stops connection breaking).
- Update deps

### CODE REVIEW

File: hotkey.js [hunk 1]
```diff
@@ -117,6 +133,7 @@ class instance extends InstanceBase {
 			console.log('connected')
 			clearInterval(this.intervalConnect)
 			this.retrying = false
+			this.startKATimer
```
Comment (jswalden) [inline hotkey.js:136]: Missing parens?

Comment (phillipivan) [reply hotkey.js]: Fixed.

**Comment by JeffreyDavidsz** [comment]
thank you, I'll make an edit later so the connection gets log in the listener also

**Comment by phillipivan** [comment]
No worries. I think it was appearing in the logs as 'wrong type'. If you're editing the app, perhaps it's worth defining a specific Keep Alive / Ping message?

Thanks for merging this so quickly, often these benign looking changes are the difference between modules being useable or not for us.

**Comment by JeffreyDavidsz** [comment]
Yea for now there is a heartbeat message in the log. Depending on my time I'll revert it to a fixed place so the log doesn't get clouded. I'm rewriting the app because of some old stuff, hopefully I can do a new release within the next week

**Comment by phillipivan** [comment]
Ahh, I see you added a `keepAlive` type, makes more sense now.

**Pull request merged**